ngrok/provider.go

```diff
@@ -28,9 +28,11 @@ func Provider() *schema.Provider {
 		ResourcesMap: map[string]*schema.Resource{
 			"ngrok_agent_ingress":             resourceAgentIngresses(),
 			"ngrok_api_key":                   resourceAPIKeys(),
+			"ngrok_failover_backend":          resourceFailoverBackends(),
+			"ngrok_http_response_backend":     resourceHTTPResponseBackends(),
+			"ngrok_tunnel_group_backend":      resourceTunnelGroupBackends(),
 			"ngrok_certificate_authority":     resourceCertificateAuthorities(),
 			"ngrok_credential":                resourceCredentials(),
-			"ngrok_endpoint_configuration":    resourceEndpointConfigurations(),
```

Comment:
Is it ok that `ngrok_endpoint_configuration` is removed?

Reply:
yeah, endpoint configuration is being merged/manged by edges now afaik.